webapp/sources/rudder/rudder-core/src/main/resources/hooks.d/policy-generation-finished/60-trigger-node-update

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
+
+# Hooks parameter are passed by environment variable:
+#
+# - RUDDER_GENERATION_DATETIME       : generation datetime: ISO-8601 YYYY-MM-ddTHH:mm:ss.sssZ date/time that identify that policy generation.
+# - RUDDER_END_GENERATION_DATETIME   : generation end time: ISO-8601 YYYY-MM-ddTHH:mm:ss.sssZ date/time when the generation ended (minus these hooks)
+# - RUDDER_NODE_IDS                  : space separated list of node id updated during the process, or the empty string
+#                                      if no nodes were updated.
+# - RUDDER_NUMBER_NODES_UPDATED      : integer >= 0; number of nodes updated (could be found by counting $RUDDER_NODE_IDS)
+# - RUDDER_ROOT_POLICY_SERVER_UPDATED: 0 if root was updated, anything else if not
+
+
+# This hook start a remote run for updated nodes.
+# This hook uses the remote-run API, so you need to open relevant port,
+# see documentation for "Trigger remote agent run" in
+# https://docs.rudder.io/reference/current/installation/requirements.html#configure-the-network
+#
+# This feature can generate a more run than expected and some load on the network. In big
+# Rudder installation (thousands of nodes), we advice to measure these metrics and disable
+# the hook if necessary.
+#
+
+# if ${TEST} is defined, property file is not read and curl command not executed
+set -e
+
+if [ -z "${TEST}" ]; then
+  source ./$(basename $0).properties
+fi
+
+MAX_NODES=${MAX_NODES:=100}
+NODE_PERCENT=${NODE_PERCENT:=100}
+RUDDER_NUMBER_NODES_UPDATED=${RUDDER_NUMBER_NODES_UPDATED:=0}
+
+if (( ${MAX_NODES} < 0 )); then
```

Comment:
this syntax doesn't exist in bash
[ ${MAX_NODES} -lt 0 ]

Reply:
what exists in bash is $(( )) but you need to put /bin/bash in the shebang

Reply:
done